### ISSUE TITLE
Accessibility - external keyboard navigation over modal does not work correctly

### ISSUE THREAD

**Issue by elless** [ionic-team/ionic#13213]
**Ionic version:**  (check one with "x")
[ ] **2.x**
[x] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[ ] bug report
[x] feature request

**Current behavior:**

The following issue occurs when using an external keyboard to navigate in an Ionic app (for accessibility).
When a modal is displayed - the navigation also goes over the elements in the parent page

**Expected behavior:**
When navigating with an Ionic app with an external keyboard - only the displayed items should be part of the navigation cycle. 

**Steps to reproduce:**
1. Create an Ionic app with tabs, open it in a phone
2. Connect phone to external keyboard
3. Turn on the "voice over"/"voice assistant"
4. Open a modal from one of the child pages of a tab.
5. Try to navigate with external keyboard over elements in the page
Result: elements in the page under the modal (that aren't visible) are also being navigated over. 

**Other information:**
Aria-hidden attribute should probably be used on pages when they aren't visible to the user - this makes them be excluded from the navigation path. 
This issue also happens with alerts, action sheets, etc.

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 6.5.0 
Ionic Framework Version: 3.5.3
Ionic CLI Version: 2.2.3
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 2.0.2
ios-deploy version: 1.9.2 
ios-sim version: 6.0.0 
OS: macOS Sierra
Node Version: v6.11.1
Xcode version: Xcode 9.0.1 Build version 9A1004
```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! Would you be able to provide a sample application via GitHub that demonstrates the issue you are having? Thanks for using Ionic!

**Comment by elless** [comment]
https://github.com/elless/IonicAccessibilityBug
Repro:
1. Connect to external keyboard and turn on "voice over"
2. Click on "Open Modal"
3. Try to navigate over the elements on the screen

Result: Elements that are part of the Home page (page that is under the modal) will also be highlighted as part of the navigation, even though they aren't shown on screen.
Expected: should only be able to reach visible elements (elements that are part of modal)

**Comment by AndreVicenteW** [comment]
Did find the solution?
I'm having a error with modal and menu
